# Numster
Numster: a novel nucleosome mapping method using a hierarchical agglomerative clustering algorithm

Boseon Byeon, Department of Neuroscience and Regenerative Medicine and Department of Neurology, Medical College of Georgia, Georgia Regents University, Augusta, GA 30912

Bo-Shiun Chen, Department of Neuroscience and Regenerative Medicine and Department of Neurology, Medical College of Georgia, Georgia Regents University, Augusta, GA 30912

Xiuwen Liu, Department of Computer Science, Florida State University, Tallahassee, FL 32306

Wei-Hua Wu, Department of Neuroscience and Regenerative Medicine and Department of Neurology, Medical College of Georgia, Georgia Regents University, Augusta, GA 30912


Chromatin structure is important for regulation of DNA-related biological events.  Nucleosome mapping by the next generation sequencing technology has become a powerful approach to study the primary structure of chromatin.  However, given the variations in sequencing tag distribution and quantity, it is challenging to map nucleosome position accurately.  In addition, most available nucleosome prediction methods were developed for analyzing sequencing reads of mononucleosomal DNA generated by micrococcal nuclease (MNase) digestion.  Here we describe a novel nucleosome positioning method, Numster, which applies a hierarchical clustering algorithm to map nucleosomes accurately from sequencing reads generated from either the MNase or the hydroxyl radical treated samples.   

SOFTWARE
The 64-bit Linux executable version of Numster and user instructions can be downloaded by clicking Linux64 above or following <a href="https://github.com/Numster/Numster/tree/master/Linux64">https://github.com/Numster/Numster/tree/master/Linux64</a>.
 
The "Numster" folder contains three files (and they each can be downloaded individually after you click Linux64 above):

1) readme.pdf

2) Numster binary software

3) testing data sets including "ho_tag.txt" and "mn_tag.txt"
 
A command "chmod +x numster" may be needed to execute the unzipped Numster file.

The C/C++ source codes of Numster can be downloaded by clicking <bf>Source</bf> above or https://github.com/Numster/Numster/tree/master/Source . 
 
INPUT DATA 
Due to their size, the input data for both testing the program and biological analysis can be downloaded from http://www.gru.edu/colleges/medicine/dnrm/faculty/wu/InputTag.zip .  
 
The "InputTag" folder contains the following files/folders: 
 
1) Readme.txt.

2) The SimulatedTag folder contains simulated tags formatted for indicated softwares.  

3) The YeastTag_Kaplan folder contains yeast MNase tags formatted for indicated softwares.  

4) The YeastTag_Shivaswamy folder contains yeast MNase sequencing tags formatted for indicated softwares.

5) The DrosophilaTag folder contains Drosophila MNase sequencing tags formatted for indicated softwares.

6) The MouseTag folder contains mouse MNase sequencing tags of chromosome I formatted for indicated softwares.

7) The HydroxylRadicalTag folder contains yeast hydroxyl radical sequencing tags formatted for indicated softwares.

8) The YeastTag_KaplanXi folder contains sequencing tags for biological analysis.   
 
 
OUTPUT DATA
The output data for both testing the program and biological analysis can be downloaded here.  
 
The "OutputNuc" folder contains the following files/folders: 
 
1) Readme.txt.

2) The SimulatedNuc folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares, as well as simulated true nucleosome positions.  

3) The YeastNuc_Kaplan folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.  

4) The YeastNuc_Shivaswamy folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.  

5) The DrosophilaNuc folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.  

6) The MouseNuc folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.  

7) The HydroxylRadicalNuc folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.  
8) The YeastNuc_KaplanXi folder contains nucleosome position and occupancy of predicated nucleosomes by indicated softwares.    
 
